common/views/components/NewsletterPromo/NewsletterPromo.js

```diff
@@ -1,87 +1,266 @@
+})`
+  display: flex;
+  flex-wrap: wrap;
+  flex: 1;
+  align-items: flex-start;
+  position: relative;
+
+  label {
+    flex: 1;
+    min-width: 300px;
+  }
+
+  ${props => props.theme.media.medium`
+    flex-wrap: nowrap;
+  `}
+`;
+
+const YellowBox = styled(Space).attrs({
+  h: { size: 'xl', properties: ['padding-left', 'padding-right'] },
+  v: { size: 'l', properties: ['padding-top', 'padding-bottom'] },
+})`
+  border: 12px solid ${props => props.theme.colors.yellow};
+
+  p {
+    max-width: 600px;
+  }
+`;
+
+const BoxInner = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+
+  ${props => props.theme.media.xlarge`
+    flex-wrap: nowrap;
   `}
 `;
 
-const NewsletterPromo = () => (
-  <div className="row bg-teal">
-    <Layout gridSizes={{ s: 12, m: 12, l: 12, xl: 12 }}>
-      <div
-        className={classNames({
-          grid: true,
-        })}
-      >
-        <div
-          className={classNames({
-            [grid({ s: 12, m: 6, l: 6, xl: 6 })]: true,
-          })}
-        >
-          <Space
-            v={{ size: 'xl', properties: ['padding-top', 'padding-bottom'] }}
-            h={{ size: 'm', properties: ['padding-left', 'padding-right'] }}
-          >
+const NewsletterPromo = () => {
+  const { altNewsletterSignupCopy } = useContext(TogglesContext);
+  const [isSubmitting, setIsSubmitting] = useState(false);
+  const [isSuccess, setIsSuccess] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  const headingText = altNewsletterSignupCopy
+    ? 'Stay in the know'
+    : `Sign up to our newsletter`;
+
+  const bodyText =
+    altNewsletterSignupCopy &&
+    'Find out what’s on, read our latest stories and get involved.';
+
+  async function handleSubmit(event) {
+    event.preventDefault();
+    setIsSubmitting(true);
+    setIsError(false);
+
+    const formEls = [...event.currentTarget.elements];
+    const data = {
+      addressbookid: formEls.find(el => el.name === 'addressbookid').value,
+      email: formEls.find(el => el.name === 'email').value,
+    };
+
+    const response = await fetch('/newsletter-signup', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(data),
+    });
+
+    const json = await response.json();
+    const { status } = json;
+
+    switch (status) {
+      case 'ContactChallenged':
+      case 'PendingOptIn':
+      case 'Subscribed':
+        setIsSuccess(true);
+        trackEvent({
+          category: 'NewsletterPromo',
+          action: 'submit email',
+        });
+        break;
+      default:
+        setIsError(true);
+
+        Raven.captureException(new Error(`Newsletter signup error: ${status}`));
+    }
+
+    setIsSubmitting(false);
+  }
+
+  return (
+    <div className="row">
+      <div className="container">
+        <div className="flex flex--h-center">
+          <div>
```

Comment:
Since the content of this updates dynamically, it should probably have an aria-live="polite" attribute

Reply:
Ah yeah. I'll add it to the `YellowBox` styled component.